src/main/java/com/greenfoxacademy/goddesstribesbackend/services/BuildingService.java

```diff
@@ -117,52 +124,23 @@ public Building createBuilding(Kingdom kingdom, String type) {
       building = null;
     }
 
+    Resource goldResource = resourceRepository.findResourceByTownhall_Kingdom_IdAndType(kingdom.getId(), ResourceTypeENUM.GOLD).get();
```

Comment:
This underline is convention for lists in the repo?

Reply:
Igen. Ezt a findResourceByTownhall_Kingdom_IdAndType() metódust nem én írom így, hanem a repositoryban a felajánlott lehetőségek közül válogatva és entereket nyomogatva így jön létre...